service/src/main/java/uk/gov/hmcts/reform/fpl/model/notify/returnedcase/ReturnedCaseTemplate.java

```diff
@@ -1,11 +1,13 @@
 package uk.gov.hmcts.reform.fpl.model.notify.returnedcase;
 
-import lombok.Getter;
-import lombok.Setter;
+import lombok.Builder;
+import lombok.Data;
 import uk.gov.hmcts.reform.fpl.model.notify.NotifyData;
 
-@Getter
-@Setter
+import java.util.Map;
+
+@Data
+@Builder
```

Comment:
Thoughts on using this instead https://projectlombok.org/features/experimental/Accessors.
So that we don't run into the annoyances described here https://tools.hmcts.net/jira/browse/FPLA-1856

Reply:
looks interesting, the only think is that builders promotes usage of immutable objects, with accessors we can nicely mimic builder syntax but immutability is the price